utilities/nodes/map_editor/README.md

```
@@ -56,3 +56,45 @@ Now that you have a perfectly closed loop in your `MAP_NAME.png`, we feed it bac
+    ros2 launch map_editor map_editor_launch.xml map_name:=MAP_NAME map_editor_mapping:=True racecar_version:=NUCNUC
+    ```
+5. After you gathered enough data, ```ctrl+c``` to both terminals (bag and launch file)
+
+Now, it's time to create the map
+
+6. **In the main container**, launch the Map Editor in **Mapping Mode** (`map_editor_mapping:=True`), and add the flag ```use_sim_time:=True```.
+    ```bash
+    ros2 launch map_editor map_editor_launch.xml map_name:=MAP_NAME map_editor_mapping:=True racecar_version:=NUCNUC use_sim_time:=True
+    ```
+7. In the terminal, in the the folder where you recorded the bag, play the bag with the clock flag:
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)